parse.y

```diff
@@ -2961,6 +2961,18 @@ rb_parser_ary_free(rb_parser_t *p, rb_parser_ary_t *ary)
                                 }
                             ;
 
+%rule f_kwarg(kw): kw
+                    {
+                        $$ = $1;
+                    /*% ripper: rb_ary_new3(1, get_value($:1)) %*/
+                    }
+                | f_kwarg(kw) <node_kw_arg> ',' kw
```

Comment:
Build failed with these errors (e.g. https://github.com/ruby/ruby/actions/runs/8872410494/job/24356719061)

```
parse.y:2969:30: parse error on value '<node_kw_arg>' (TAG)
                | f_kwarg(kw) <node_kw_arg> ',' kw
                              ^^^^^^^^^^^^^
make: *** [uncommon.mk:1075: parse.c] Error 1
```

Because Lrama grammar doesn't support `tag_opt` is not allowed in `rule_rhs` (https://github.com/ruby/lrama/blob/e2d9fd3187e96683cc1c975258fe0b74d0decfa6/parser.y#L285). It needs to allow `tag_opt` like https://github.com/ruby/lrama/blob/e2d9fd3187e96683cc1c975258fe0b74d0decfa6/parser.y#L452.